javascript/asynchronous_javascript_and_apis/project_weather_app.md

```diff
@@ -1,34 +1,34 @@
 ### Introduction
 
-Use everything we've been discussing to create a weather forecast site using the weather API from the previous lesson. You should be able to search for a specific location and toggle displaying the data in Fahrenheit or Celsius.
+Use everything we've been discussing to create a weather forecast site using the Visual Crossing API from the previous lesson. You should be able to search for a specific location and toggle displaying the data in Fahrenheit or Celsius.
 
 You should change the look of the page based on the data, maybe by changing the color of the background or by adding images that describe the weather. (You could even use the Giphy API to find appropriate weather-related gifs and display them). Feel free to use promises or async/await in your code, though you should try to become comfortable with both.
 
 <div class="lesson-note" markdown="1">
```

Comment:
Even after removing the above though, what I think this note box could be used for instead is a temporary mention that the API for the project (and previous lesson) has changed to Visual Crossing, and that it's still mostly the same concepts, just need an account with Visual Crossing instead and check its docs for exactly how to use its API and the free tier features/restrictions.

Once merged, I'll remove the note box like 2 weeks after perhaps. Just so it's not a brick wall in the face for people not expecting it. After 2-3 weeks, we shouldn't need the note anymore, I don't think.

Reply:
Essentially replace the note box's contents with something like this (very rough wording suggestion, feel free to amend as you see fit since it's probably unnecessarily wordy):

```md
#### Change of weather API to Visual Crossing

We have recently replaced the weather API used in the previous lesson and this project with Visual Crossing's API, due to intermittent but somewhat frequent issues with the previous API.

The main concepts will still be the same. Make a free account with them if you have not already done so, and check their API documentation for exactly how to query for the data you want. Remember to check out [Visual Crossing's query builder](https://www.visualcrossing.com/weather/weather-data-services)!
```

Reply:
Will do